### ISSUE TITLE
AVT3 fail to open link in data table with JAWS active

### ISSUE THREAD

**Issue by jacklotusho** [carbon-design-system/carbon#279]
AVT3 fail to open link in data table with JAWS active, but it is working fine AVT2(tab and Enter without JAWS)

When tab to link in data table, it supposes to open link instead of make checkbox selected when JAWS active. (Below is screen when before enter is hit.)
![image](https://user-images.githubusercontent.com/12465435/29275845-84b3a6b4-80da-11e7-963a-bd7f28ed5557.png)

**Comment by tw15egan** [comment]
This needs to be fixed by the product team that has implemented this. We only support an overflow menu as the last item.
